GVRf/Framework/framework/src/main/java/org/gearvrf/GVRScene.java

```diff
@@ -128,6 +130,14 @@ public void clear() {
     }
 
     /**
+     * Get the root of the scene hierarchy.
+     * @return top level scene object.
+     */
+    public GVRSceneObject getRoot() {
+        return mSceneRoot;
+    }
+    
+    /**
      * The top-level scene objects.
      * 
      * @return A read-only list containing all the 'root' scene objects (those
```

Comment:
Javadoc probably needs update since now we have a root sceneObject and not a list of "root objects".